agents/PCS-Large-Exhange-Rate-Update/src/constants.ts

```diff
@@ -0,0 +1,29 @@
+import { BigNumber } from "ethers";
+
+const STKBNB_POOL_ADDRESS = "0xaa2527ff1893e0d40d4a454623d362b79e8bb7f1";
+const STKBNB_TOKEN_ADDRESS = "0xc2E9d07F66A89c44062459A47a0D2Dc038E4fb16";
+const WBNB_TOKEN_ADDRESS = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+
+const ER_THRESHOLD = Number("0"); // percent change in exchange rate threshold from previous exchange rate.
```

Comment:
Shouldn't this need to be set 1%  as mentioned in README?

Reply:
Done